docs/source/index.rst

```diff
@@ -7,39 +7,47 @@ Dialog Flow Framework Documentation
+for a wide range of applications, including chatbots, virtual assistants, and other interactive systems.
+It allows developers to easily create and manage complex dialog flows, integrate with natural language processing (NLP) tools,
+and handle user input in a flexible and efficient manner. Additionally, the framework is highly customizable,
+allowing developers to easily adapt it to their specific needs and requirements. 
 
 Getting started
 ---------------
 
 Installation
 ~~~~~~~~~~~~
 
-`DFF` can be installed via ``pip``:
+`DFF` can be easily installed on your system using the ``pip`` package manager:
 
 .. code-block:: console
    
    pip install dff
 
-and supports Python 3.7 and newer. You can also download the library from `Github <https://github.com/deeppavlov/dialog_flow_framework>`_:
+This framework is compatible with Python 3.7 and newer versions.
+Note that if you are going to use one of the database backends, you will have to specify an extra or install the corresponding requirements yourself.
+Additionally, you also have the option to download the source code directly from the
+`GitHub <https://github.com/deeppavlov/dialog_flow_framework>`_ repository using the commands:
 
 .. code-block:: console
 
    git clone https://github.com/deeppavlov/dialog_flow_framework.git
    cd dialog_flow_framework
-   make venv
 
-The last command will set up all requirements. If requirements need to be updated, use the command ``make clean`` before ``make venv``.
+Once you are in the directory, you can run the command ``make venv`` to set up all the necessary requirements for the library.
+If you need to update the requirements in the future, use the command ``make clean`` to ensure that the new requirements are properly installed.
 
 Key concepts
 ~~~~~~~~~~~~
 
-`DFF` allows you to write conversational services. The service is written by defining a special dialog graph
-that describes the behavior of the dialog service. `DFF` offers a Specialized Language (`DSL`) for quickly writing dialog graphs.
-You can use it to write chatbots for social networks, call centers, websites, writing skills for Amazon Alexa, etc.
+`DFF`` is a powerful tool for creating conversational services.
+It allows developers to easily write and manage dialog systems by defining a special dialog graph that describes the behavior of the service.
+`DFF`` offers a specialized language (`DSL`) for quickly writing dialog graphs, making it easy for developers to create chatbots for a wide
+range of applications such as social networks, call centers, websites, skills for Amazon Alexa, etc.
 
-`DFF` has the following important concepts:
+`DFF` has several important concepts:
 
 **Script**: First of all, to create a dialog agent it is necessary to create a dialog (:py:class:`~dff.script.core.script.Script`).
 A dialog `script` is a dictionary, where keys correspond to different `flows`. A script can contain multiple `scripts`, what is needed in order to divide
```

Comment:
will be better if we notice that this multiple `scripts` are `flows`, actualy


`A script can contain multiple `scripts`, what is needed in order to divide`